src/jit/lowerarmarch.cpp

```diff
@@ -326,6 +326,20 @@ void Lowering::LowerBlockStore(GenTreeBlk* blkNode)
             if (!blkNode->OperIs(GT_STORE_DYN_BLK) && (size <= CPBLK_UNROLL_LIMIT))
```

Comment:
while you are here could you please change the condition to `if (!blkNode->OperIs(GT_STORE_BLK) && (size <= CPBLK_UNROLL_LIMIT))`?

Reply:
Sure! But it can probably wait until the next PR (immediately after this one). Let's not kill the CI more than it's needed. :grin:

Looks like it can be even better
```
if (obj) {
}
else if (blk && unroll) {
}
else {
}
```
The less nesting the better.